server/graphql/v1/mutations/notifications.js

```diff
@@ -0,0 +1,88 @@
+import Promise from 'bluebird';
+import { values } from 'lodash';
+import models, { Op } from '../../../models';
+import activities from '../../../constants/activities';
+import { channels } from '../../../constants';
+import { Forbidden } from '../../errors';
+
+const NotificationPermissionError = new Forbidden({
+  message: "This notification does not exist or you don't have the permission to edit it.",
+});
+
+export async function editNotifications(args, remoteUser) {
+  if (!remoteUser) {
```

Comment:
We also need to check that `remoteUser` is admin of collective `args.id`. See https://github.com/opencollective/opencollective-api/blob/32f473bdbd17056bf8dbef7e6e6c392c32fa8747/server/graphql/v1/mutations/collectives.js#L691 for an example